### DISCUSSION TITLE
Client is Empty

### DISCUSSION THREAD

**Discussion by jempee** [chadwpalm/Lumunarr#16] [Q&A]
Dear all,

i'm a newbie and i'm searching some help.

I've installed hueplex on my NAS Synology with docker.
HuePlex web site is on.
I'm connected on it, my Bridge is linked, My Plex Annual Pass is connected to.
On my plex server, the webhook tab is filled correctly.
On server Tab, I can retrieve on my lamps.
However on client Tab, i only have the possibility to Add a client. When I try, i get a blank screen on my chrome brower.

If you have any idea to help me, because when i read a movie there is no action on my lamps :-)
Sure, I must have missed something, but what...

thanks in advance for your help,

BR,
Dominique

**Comment by chadwpalm** [comment]
Hi Dominique,

It sounds like you have everything set up properly, so I am curious as to why the screen goes blank when trying to add a client.

Since you are setting this up on Docker, all of the app's logs will be found in the Docker logs for the container.

You said that you set this up on Synology (currently how mine is set up as well), I'm curious which method you used to set up the container. Did you SSH in and use "docker run" or docker-compose, or did you set it all up in Synology's Container Manager?

The easiest way to get the logs would be from the Container Manager in DSM (could still be called Docker if you are on an older DSM). From there, go to the Container tab, double click on the hueplex container, then go to the logs tab. Ideally, you will want to look at the logs at the time you tried to add a client. I believe something is erroring on the backend and the information is not making it back to the frontend GUI.

Thanks,
Chad

**Comment by Lothouf** [comment]
Hi, 

I have exactly the same problem, blank screen when i click on "add client"
here the log: 
![image](https://github.com/chadwpalm/HuePlex/assets/143877165/84503787-856c-4423-9bf3-5d445266eb05)

Thx for your help

**Reply by chadwpalm** [reply]
Someone else (not the person above) was having this problem and filed an issue. We resolved it by adding "--network host" to the command for starting the container. Perhaps try and see if this fix works for you.

https://github.com/chadwpalm/HuePlex/issues/17#issuecomment-1666588543
